core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -583,6 +583,8 @@ public interface MetricsProducer {
 
   String METRICS_COMPACTOR_PREFIX = "accumulo.compactor.";
   String METRICS_COMPACTOR_MAJC_STUCK = METRICS_COMPACTOR_PREFIX + "majc.stuck";
+  String METRICS_COMPACTOR_ENTRIES_READ = METRICS_COMPACTOR_PREFIX + "entries.read";
```

Comment:
These are good names for elasticity where there are no longer compaction running in tservers.

Reply:
Do you think its fine to leave this as the name for 2.1 and main where tservers could be running the compactions? Or should we change the name to something more agnostic of where the compaction is running for those?

Reply:
I think the name is fine. The name is not optimal for 2.1, but its great for later releases.